(node:3360044) Warning: Accessing non-existent property 'padLevels' of module exports inside circular dependency
(Use `node --trace-warnings ...` to show where the warning was created)
Started with the following options:
{
  debug: true,
  mitmIp: '10.0.3.1',
  autoAccess: true,
  autoAccessCache: 5000,
  maxAttemptsPerConnection: 6,
  sshServerIdentifier: 'SSH-2.0-OpenSSH_6.6.1p1 Ubuntu-2ubuntu2',
  containerMountPathPrefix: '/var/lib/lxc',
  containerMountPathSuffix: 'rootfs/',
  loggingAttackerStreams: '/home/student/MITM/logs/session_streams',
  loggingAuthenticationAttempts: '/home/student/MITM/logs/authentication_attempts',
  loggingLogins: '/home/student/MITM/logs/logins',
  loggingLogouts: '/home/student/MITM/logs/logouts',
  loggingKeystrokes: '/home/student/MITM/logs/keystrokes',
  containerName: 'nobanner-nosession-40662',
  containerIp: '10.0.3.138',
  mitmPort: 40662,
  autoAccessFixed: 1
}
2024-11-06 09:58:08.002 - [Info] MITM Version: 2
2024-11-06 09:58:08.014 - [Info] Auto Access Enabled: true
2024-11-06 09:58:08.015 - [Debug] [Init] Auto Access Theshold Achieved: false
2024-11-06 09:58:08.027 - [Info] SSH man-in-the-middle server for 10.0.3.138 listening on 10.0.3.1:40662
2024-11-06 09:59:15.559 - [Debug] [Connection] Attacker connected: 175.100.24.139 | Client Identification: SSH-2.0-libssh_0.9.6
2024-11-06 09:59:17.307 - [Debug] [Auth] Attacker 175.100.24.139 trying to authenticate with 'password'
2024-11-06 09:59:17.308 - [Debug] [Auto Access] Attacker: 175.100.24.139, Threshold: 1, Attempts: 1
2024-11-06 09:59:17.308 - [Debug] [Auto Access] Compromising the honeypot
2024-11-06 09:59:17.309 - [Debug] [Auto Access] Adding the following credentials: 'q:qpass'
2024-11-06 09:59:17.693 - [Debug] [Auto Access] Auto-access is now disabled for the remainder of this MITM server instance
2024-11-06 09:59:17.696 - [Debug] [Auth] Valid credentials - Password Authentication
2024-11-06 09:59:17.696 - [Debug] [LXC] Attempting to connect to the honeypot: 10.0.3.138
2024-11-06 09:59:17.789 - [Debug] [LXC-Auth] Attacker authenticated and is inside container
2024-11-06 09:59:18.994 - [Debug] [EXEC] Noninteractive mode attacker command: cd ~; chattr -ia .ssh; lockr -ia .ssh
2024-11-06 09:59:20.789 - [Debug] [Connection] Attacker connected: 175.100.24.139 | Client Identification: SSH-2.0-libssh_0.9.6
2024-11-06 09:59:22.504 - [Debug] [Auth] Attacker 175.100.24.139 trying to authenticate with 'password'
2024-11-06 09:59:22.504 - [Debug] [Auth] Attacker authentication error: Invalid credentials - User does not exist
2024-11-06 09:59:22.506 - [Debug] [Auth] Attacker: 175.100.24.139 has so far made 1 attempts. Remaining: 5 attempts
2024-11-06 09:59:23.099 - [Debug] [Connection] Attacker closed the connection
2024-11-06 09:59:23.387 - [Debug] [Connection] Attacker connected: 175.100.24.139 | Client Identification: SSH-2.0-libssh_0.9.6
2024-11-06 09:59:23.672 - [Debug] [Connection] Attacker connected: 195.178.110.89 | Client Identification: SSH-2.0-Go
2024-11-06 09:59:24.246 - [Debug] [Auth] Attacker 195.178.110.89 trying to authenticate with 'none'
2024-11-06 09:59:24.246 - [Debug] [Auth] Attacker authentication error: No authentication method provided
2024-11-06 09:59:24.462 - [Debug] [Auth] Attacker 195.178.110.89 trying to authenticate with 'password'
2024-11-06 09:59:24.462 - [Debug] [Auth] Attacker authentication error: Invalid credentials - Container user is disabled
2024-11-06 09:59:24.463 - [Debug] [Auth] Attacker: 195.178.110.89 has so far made 1 attempts. Remaining: 5 attempts
{"level":"error","message":"Forever detected script was killed by signal: SIGKILL"}
